pyproject.toml

```diff
@@ -88,7 +88,7 @@ repository = "https://github.com/commaai/openpilot"
 documentation = "https://docs.comma.ai"
 
 [tool.poetry.dependencies]
-python = "~3.11"
+python = ">=3.11, <3.13"
```

Comment:
can we do <4? I always find it annoying when projects do this

Reply:
Do you mean ">=3.11, <4.0" ?

Reply:
yea, I think that's what most projects do

Reply:
poetry does not seem to work with non-existing version (even <3.14 does not work)